.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md
```

Comment:
- [ ] Good job adding the `test.md` file to `.gitignore`. However, the `test.md` file appeared in your repository.

![test-md](https://user-images.githubusercontent.com/36542986/131885850-07732a9d-8ca0-47e9-81fe-0217f462f34b.jpg)
This is because you probably added the `test.md` after creating it. You need to add the file/directory to be ignored in `.gitignore` before actually adding it to your repository.

You can refer to this [Stackoverflow answer](https://stackoverflow.com/a/45400404) for further elaboration.